ext/mysqli/tests/functions/mysqli_get_server_info.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+function test: mysqli_get_server_info
```

Comment:
Why not delete ext\mysqli\tests\mysqli_get_server_info.phpt?

Reply:
Could do